boards/lilygo/ttgo_t8s3/doc/index.rst

```diff
@@ -0,0 +1,238 @@
+.. _ttgo_t8s3:
+
+Lilygo TTGO T8-S3
+#################
+
+Overview
+********
+
+Lilygo TTGO T8-S3 is an IoT mini development board based on the
+Espressif ESP32-S3 WiFi/Bluetooth dual-mode chip.
+
+It features the following integrated components:
+
+- ESP32-S3 chip (240MHz dual core, Bluetooth LE, Wi-Fi)
+- on board antenna and IPEX connector
+- USB-C connector for power and communication
+- MX 1.25mm 2-pin battery connector
+- JST SH 1.0mm 4-pin UART connector
+
+.. figure:: img/ttgo_t8s3.webp
+   :align: center
+   :alt: TTGO T8-S3
+
+   Lilygo TTGO T8-S3
+
+Functional Description
+**********************
+This board is based on the ESP32-S3 with 16MB of flash, WiFi and BLE support. It
+has an USB-C port for programming and debugging, integrated battery charging
+and an on-board antenna. The fitted U.FL external antenna connector can be
+enabled by moving a 0-ohm resistor.
+
+Connections and IOs
+===================
+
+The TTGO T8-S3 board configuration supports the following hardware features:
```

Comment:
replace with board name used by west with \`\` around it and refer it to as board target

Reply:
Hi nordicjm,

should I replace all occurences of TTGO T8-S3 by ``ttgo_t8s3``?
Then I should probably update the ttgo_lora32 and ttgo_t8c3 documents too?

Reply:
I updated the documentation, please let me know if it's better now.
When you approve it, I'll update the t8c3 and lora32 too.